loader/include/Geode/utils/web.hpp

```diff
@@ -82,7 +82,7 @@ namespace geode::utils::web {
         Result<> into(std::filesystem::path const& path) const;
 
         std::vector<std::string> headers() const;
-        std::optional<std::string> header(std::string_view name) const;
+        std::optional<std::string> header(std::string_view name, int index = 0) const;

```

Comment:
If we're doing this change imo this should return the full vector

Reply:
I did it in this way to make it backwards compatible but yeah that sounds like a better solution

Reply:
personally id rather have a new method, since for 99% of use cases the header value you want is just a single value

Reply:
ok, header(sv) returns a single header like it used to after a change I made, and I added headersWithName(sv) that returns the whole vector instead